build.sbt

```diff
@@ -122,3 +122,13 @@ lazy val bench = project
       "jsdom" -> "9.9.0"
     ),
   )
+
+
+lazy val root = project
+  .in(file("."))
+  .settings(
+    name := "outwatch-root",
+    skip in publish := true,
+  )
+  .dependsOn(outwatch, bench)
```

Comment:
I would have thought `aggregate` is enough. Why do we need `dependsOn` for outwatch as well as bench?

Reply:
You're right, I think just aggregate on both is good. I did it this way to make sure bench gets compiled, but bench is already blocked for publishing. Will update.